errors from the pre-commit:
```
In file included from /home/runner/work/llvm-project/llvm-project/libc/src/wchar/wcstok.cpp:13:
/home/runner/work/llvm-project/llvm-project/libc/src/wchar/wchar_utils.h:23:10: error: cannot initialize return object of type 'wchar_t *' with an rvalue of type 'const wchar_t *'
   23 |   return (*s == c) ? s : nullptr;
      |          ^~~~~~~~~~~~~~~~~~~~~~~
/home/runner/work/llvm-project/llvm-project/libc/src/wchar/wchar_utils.h:32:19: error: no matching function for call to 'wcschr'
   32 |     bool in_set = wcschr(s1[i], s2);
      |                   ^~~~~~
/home/runner/work/llvm-project/llvm-project/libc/src/wchar/wchar_utils.h:20:29: note: candidate function not viable: no known conversion from 'const wchar_t' to 'const wchar_t *' for 1st argument; take the address of the argument with &
   20 | LIBC_INLINE static wchar_t *wcschr(const wchar_t *s, wchar_t c) {
      |                             ^      ~~~~~~~~~~~~~~~~
   21 |   for (; *s && *s != c; ++s)
   22 |     ;
   23 |   return (*s == c) ? s : nullptr;
   24 | }
   ```

--- a/libc/src/wchar/wchar_utils.h
+++ b/libc/src/wchar/wchar_utils.h
@@ -17,7 +17,7 @@
 namespace LIBC_NAMESPACE_DECL {
 namespace internal {
 
-LIBC_INLINE static wchar_t *wcschr(const wchar_t *s, wchar_t c) {
+LIBC_INLINE static const wchar_t *wcschr(const wchar_t *s, wchar_t c) {
   for (; *s && *s != c; ++s)
     ;
   return (*s == c) ? s : nullptr;
@@ -29,7 +29,7 @@ LIBC_INLINE static size_t wcsspn(const wchar_t *s1, const wchar_t *s2,
                                  bool not_match_set) {
   size_t i = 0;
   for (; s1[i]; ++i) {
-    bool in_set = wcschr(s1[i], s2);
+    bool in_set = internal::wcschr(s2, s1[i]);
     if (in_set == not_match_set)
       return i;
   }

--- a/libc/src/wchar/wcstok.cpp
+++ b/libc/src/wchar/wcstok.cpp
@@ -24,14 +24,13 @@ LLVM_LIBC_FUNCTION(wchar_t *, wcstok,
     str = *context;
   }
 
-  wchar_t *tok_start, *tok_end;
-  for (tok_start = str; *tok_start != L'\0' && internal::wcschr(delims, *tok_start);
-       ++tok_start)
-    ;
+  wchar_t *tok_start = str;
+  while (*tok_start != L'\0' && internal::wcschr(delims, *tok_start))
+    ++tok_start;
 
-  for (tok_end = tok_start; *tok_end != L'\0' && !internal::wcschr(delims, *tok_end);
-       ++tok_end)
-    ;
+  wchar_t *tok_end = tok_start;
+  while (*tok_end != L'\0' && !internal::wcschr(delims, *tok_end))
+    ++tok_end;
 
   if (*tok_end != L'\0') {
     *tok_end = L'\0';

--- a/libc/src/wchar/wcstok.cpp
+++ b/libc/src/wchar/wcstok.cpp
@@ -25,11 +25,11 @@ LLVM_LIBC_FUNCTION(wchar_t *, wcstok,
   }
 
   wchar_t *tok_start, *tok_end;
-  for (tok_start = str; *tok_start != L'\0' && wcschr(delims, *tok_start);
+  for (tok_start = str; *tok_start != L'\0' && internal::wcschr(delims, *tok_start);
        ++tok_start)
     ;
 
-  for (tok_end = tok_start; *tok_end != L'\0' && !wcschr(delims, *tok_end);
+  for (tok_end = tok_start; *tok_end != L'\0' && !internal::wcschr(delims, *tok_end);
        ++tok_end)
     ;
 

